site/cypress/e2e/Cypress-Gradeable/grade_inquiries.spec.js

```diff
@@ -0,0 +1,40 @@
+describe('Test cases revolving around grade inquires', () => {
+    beforeEach(() => {
+        cy.visit(['sample']);
+        cy.login();
+    });
+    const setGradeInquiriesForGradeable = (gradeableId, date = null, allowed = true) => {
+        cy.get(`#${gradeableId} .fa-pencil-alt`).click();
+
+        if (allowed) {
+            cy.get('[data-testid="yes-grade-inquiry-allowed"]').click();
+        }
+        else {
+            cy.get('[data-testid="no_grade_inquiry_allowed"]').click();
+        }
+
+        if (date) {
+            cy.contains('Dates').click();
+            cy.get('[data-testid="date_grade_inquiry_due"]').click();
+
+            cy.get('[data-testid="date_grade_inquiry_due"]').should('be.visible');
+
+            cy.get('[data-testid="date_grade_inquiry_due"]').clear().type(date, { parseSpecialCharSequences: false, force: true });
+        }
+
+        cy.get('#nav-sidebar-submitty').click();
+    };
+
+    it('should test normal submission grade inquiry panel', () => {
+        const gradeableId = 'grades_released_homework';
+        const gradeInquiryDeadlineDate = '9998-01-01 00:00:00';
+        setGradeInquiriesForGradeable(gradeableId, gradeInquiryDeadlineDate, true);
+
+        cy.get(`#${gradeableId} .btn-nav-grade`).click();
+        cy.get('[data-testid="view-sections"]').click();
+        cy.get(':nth-child(4) > :nth-child(1) > :nth-child(8) > [data-testid="grade-button"]').click();
```

Comment:
Is there a way to select this without chaining selectors?

Reply:
> Is there a way to select this without chaining selectors?

this could be possible case if we are clicking on first grade-buttton and i tried it locally it works fine !

```js
cy.get('[data-testid="grade-button"]').first().click();
```

<img width="1792" alt="Screenshot 2024-04-04 at 8 34 57 PM" src="https://github.com/Submitty/Submitty/assets/119070053/7aa13887-8d1e-4a71-8923-2e04ae74036b">

<img width="1097" alt="Screenshot 2024-04-04 at 8 34 46 PM" src="https://github.com/Submitty/Submitty/assets/119070053/258eb597-63ce-4193-96b8-2455f7613a70">